dev/setup-clowder.sh

```diff
@@ -149,10 +149,10 @@ bonfire process advisor -n test --no-get-dependencies | oc apply -f - -n test
 delete_clowdapp_dependencies advisor-backend
 wait_for_pod_to_be_running app=advisor-backend,service=db
 
-# setup xjoin.v2 resources, remove v1 resource
```

Comment:
please leave these un-commented. The primary purpose of this script is for xjoin.v2 development.

Reply:
Done!